log.txt

```diff
@@ -32,3 +32,20 @@ for (const property in carolinaWren) {
     Binomial name: Thyrothorus ludovicianus
     Family: Wrens
     Habitat: Dense forest, farm edges, and suburban areas */
+
+1.3 - Concept: Calculating the sum of an array using the forEach() method, Language/Framework: JavaScript
+
+TIL... You can use the JavaScript forEach() method to find the sum of an array with numbers. The arr.forEach() method will call the provided function once for each element of the array. (GeeksforGeeks - https://www.geeksforgeeks.org/how-to-find-the-sum-of-all-elements-of-a-given-array-in-javascript/)
+
+let triangleSides = [1, 1, 1];
+
+function trianglePerimeter(arr) {
+    let sum = 0;
+    arr.forEach(x => {
+        sum += x;
+    })
+    return sum;
+}
```

Comment:
Nice work, keep going. 

Another way to calculate sum of an array number is to use  the javascript array method
```javascript
array.reduce()
```
Reference: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/reduce